weblab/static/js/experiment.js

```diff
@@ -13,6 +13,39 @@ var versions = {},
   converter = new showdown.Converter(),
   filesTable = {};
 
+function initVersionList() {
+  // Comparing experiments click events
+  var $exp_list = $(doc.version.experimentpartners).children("ul");
```

Comment:
@jonc125 this line errors with `Cannot read property 'version' of undefined` Any thoughts on why that might be, would be appreciated please?

Reply:
The `doc` variable has not been set (it would have been defined somewhere further up in the original `init` function). Looking at the template, you could probably change it to
```js
  var $exp_list = $("#entityversionlist_content").children("ul");
```
to pick out the div that contains the list of versions. jQuery makes selecting elements by id easier than the legacy javascript code!